modules/users/client/controllers/authentication.client.controller.js

```diff
@@ -31,7 +31,7 @@ angular.module('users').controller('AuthenticationController', ['$scope', '$stat
 
         // And redirect to the previous or home page
        // $state.go($state.previous.state.name || 'home', $state.previous.params);
-       $state.go('home', $state.previous.params);
+       $state.go('chat', $state.previous.params);
```

Comment:
I wanted you to create another pull request for this

Reply:
@Bakushin10 you can go later and remove the files and retouch the redirect. I just want everyone to approved the new flow first.